loopdb/sqlc/queries/static_address_deposits.sql

```diff
@@ -0,0 +1,66 @@
+-- name: CreateDeposit :exec
+INSERT INTO deposits (
+    deposit_id,
+    tx_hash,
+    out_index,
+    amount,
+    confirmation_height,
+    timeout_sweep_pk_script,
+    expiry_sweep_txid
+) VALUES (
+             $1,
+             $2,
+             $3,
+             $4,
+             $5,
+             $6,
+             $7
+         );
+
+-- name: UpdateDeposit :exec
```

Comment:
It's simpler to use just one `UpdateDeposit` function but perhaps worth considering a more granular approach where depending on the state change we only update the confirmation height or the expiry txid. WDYT? Obviously this would also mean a slight change in how the FSM works and I could also argue its a bit overly cautious.